GRIB2_Template_4_127_ProductDefinitionTemplate_en.csv

```diff
@@ -0,0 +1,55 @@
+"Individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",69,Second,,,,,Operational
+"Individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",70,n - number of time range specifications describing the time intervals used to calculate the statistically processed field,,,,,Operational
+"Individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",71-74,Total number of data values missing in statistical process,,,,,Operational
+"Individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",,75-86 Specification of the outermost (or only) time range over which statistical processing is done,,,,,Operational
+"Individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",75,Statistical process used to calculate the processed field from the field at each time increment during the time range,(see Code table 4.10),,4.1,,Operational
+"Individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",76,Type of time increment between successive fields used in the statistical processing,(see Code table 4.11),,4.11,,Operational
+"Individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",77,Indicator of unit of time for time range over which statistical processing is done,(see Code table 4.4),,4.4,,Operational
+"Individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",78-81,"Length of the time range over which statistical processing is done, in units defined by the previous octet",,,,,Operational
+"Individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",82,Indicator of unit of time for the increment between the successive fields used,(see Code table 4.4),,4.4,,Operational
+"Individual ensemble forecast, control and perturbed, at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",83-86,"Time increment between successive fields, in units defined by the previous octet",(see Notes 3 and 4),"7,99",,,Operational
```

Comment:
notes don't look correct

7,"Above normal, near normal and below normal are defined as three equiprobable categories based on climatology at each point over the geographical area covered by the grid. The type and methodology of the reference climatology are unspecified and should be documented concurrently by the data producer."

99,"The sea-ice level represents a sea-ice model level for which the depth is not constant across the model domains. The depth in metres of the level is provided by another GRIB message with the parameter “sea-ice thickness” with discipline 10, category 2 and parameter number 1."